pkg/apis/postgres-operator.crunchydata.com/v1beta1/pgadmin_types.go

```diff
@@ -52,6 +52,12 @@ type PGAdminPodSpec struct {
 	// +optional
 	Config PGAdminConfiguration `json:"config,omitempty"`
 
+	// The Secret containing the pgAdmin value for the LDAP_BIND_PASSWORD
+	// configuration setting which is used for the pgAdmin login user when LDAP
+	// authentication is enabled.
```

Comment:
⛏️ Not all LDAP situations require this setting. I think we can keep it simple and link out to the docs.

```suggestion
	// A Secret containing the value for the LDAP_BIND_PASSWORD setting.
	// More info: https://www.pgadmin.org/docs/pgadmin4/latest/ldap.html
```

Reply:
:+1: works for me, updated.